Set MySQL 5.7 as default tab for Aurora (#25680)
DataDog/documentation: content/en/database_monitoring/setup_mysql/aurora.md

Configure the following in the [DB cluster parameter group][3] and then **restart the server** for the settings to take effect:

{{< tabs >}}
{{% tab "MySQL ≥ 5.7" %}}
| Parameter | Value | Description |
| --- | --- | --- |
| `performance_schema` | `1` | Required. Enables the [Performance Schema][1]. |
| <code style="word-break:break-all;">performance_schema_consumer_events_statements_current</code> | `1` | Required. Enables monitoring of currently running queries. |
| <code style="word-break:break-all;">performance-schema-consumer-events-waits-current</code> | `ON` | Required. Enables the collection of wait events. |
| <code style="word-break:break-all;">performance_schema_consumer_events_statements_history</code> | `1` | Optional. Enables tracking recent query history per thread. If enabled it increases the likelihood of capturing execution details from infrequent queries. |
| <code style="word-break:break-all;">performance_schema_consumer_events_statements_history_long</code> | `1` | Optional. Enables tracking of a larger number of recent queries across all threads. If enabled it increases the likelihood of capturing execution details from infrequent queries. |

> | <code style="word-break:break-all;">performance_schema_max_digest_length</code> | `4096` | Increases the size of SQL digest text in `events_statements_*` tables. If left at the default value then queries longer than `1024` characters will not be collected. |

Reviewer: ```suggestion
| <code style="word-break:break-all;">performance_schema_max_digest_length</code> | `4096` | Increases the size of SQL digest text in `events_statements_*` tables. If left at the default value then queries longer than `1024` characters are not collected. |
```
